test/clojure/nrepl/describe_test.clj

```diff
@@ -1,18 +1,20 @@
 (ns nrepl.describe-test
   {:author "Chas Emerick"}
-  (:require
-   [clojure.test :refer :all]
-   [nrepl.core :as nrepl]
-   [nrepl.core-test :refer [def-repl-test repl-server-fixture project-base-dir]]
-   [nrepl.middleware :as middleware]
-   [nrepl.version :as version]))
+  (:require [clojure.test :refer [is testing use-fixtures]]
+            [nrepl.core :as nrepl]
+            [nrepl.core-test :refer [def-repl-test repl-server-fixture]]
+            [nrepl.middleware :as middleware]
+            [nrepl.server :as server]
+            [nrepl.version :as version]))
 
 (use-fixtures :once repl-server-fixture)
 
 (def ^{:private true} op-names
```

Comment:
This can even be some public API somewhere - it's useful to be able to tell apart built-in from third-party ops. I needed to do this in the docs generator of `cider-nrepl` (for instance).

Reply:
This should be just "the list of ops provided by default middleware"? It feels reasonably safe.

Are you saying other libs have a dependency on `describe-test`?

Reply:
See https://github.com/clojure-emacs/cider-nrepl/blob/master/maint/cider/nrepl/impl/docs.clj#L41

Reply:
Ah I see. So sounds like we should add `built-in-ops` to `nrepl.middleware`?